pykeybasebot/cli.py

```diff
@@ -5,7 +5,7 @@
 
 from .kbevent import KbEvent
 
-KEYBASE_TIMEOUT_MS = 2000
+KEYBASE_TIMEOUT_MS = 10000
```

Comment:
We probably want to have a bigger discussion about this, this is just a placeholder for the tests to pass.

Reply:
with the other changes, do we still need this change?